Makefile

```diff
@@ -51,9 +62,11 @@ deploy: generate ## Deploy controller in the configured Kubernetes cluster in ~/
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
 .PHONY: generate
-generate: ## Generate manifests e.g. CRD, RBAC etc.
+generate: $(testbin_created) ## Generate manifests e.g. CRD, RBAC etc.
 	@CRD_ROOT_DIR="$(CRD_ROOT_DIR)" go generate -tags=generate generate.go
 	@rm config/*.yaml
+	@cp -r config/crd/apiextensions.k8s.io $(TESTBIN_DIR)/
+	@rm $(TESTBIN_DIR)/apiextensions.k8s.io/{v1,v1beta1}/kustomiz*.yaml || true
```

Comment:
I recommend restructuring the CRD dirs a bit:
```
/config/crd/apiextensions.k8s.io/{v1,v1beta1}/kustomize.yaml
/config/crd/apiextensions.k8s.io/{v1,v1beta1}/bases/backup.appuio.ch_*.yaml
```
So we can tell envtest to search CRDs in `/config/crd/apiextensions.k8s.io/v1/bases`, where only CRD yaml files are stored. That way we don't need to copy files around.